<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListForm</class>
 <widget class="QWidget" name="ListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Способ формирования общего списка</string>
   </property>
   <widget class="QPushButton" name="uploadApplyPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="fileRadioButton">
       <property name="text">
        <string>Загрузить список из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="writeRadioButton">
       <property name="text">
        <string>Ввести список с клавиатуры</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>541</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Работа со списками</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Общий список</string>
    </property>
    <widget class="QListWidget" name="generalListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>231</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="comeToNewListPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенести в новый список</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addToGenListPushButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delFromGenListPushButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Новый список</string>
    </property>
    <widget class="QListWidget" name="newListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>541</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Запись в файл</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="saveGeneralListPushButton">
       <property name="text">
        <string>Сохранить общий список</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveNewListPushButton">
       <property name="text">
        <string>Сохранить новый список</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
